app/src/main/res/layout/activity_presentation.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+xmlns:android="http://schemas.android.com/apk/res/android"
+xmlns:tools="http://schemas.android.com/tools"
+android:layout_width="match_parent"
+android:layout_height="match_parent"
+android:orientation="vertical"
+tools:context=".activity.MainActivity"
+android:gravity="center"
+android:paddingTop="@dimen/padding_top_enter"
+android:background="@color/background_enter" >
+
+<ImageView
+    android:id="@+id/logo"
+    android:layout_width="300sp"
```

Comment:
1-. Please no hardcode, use dimens.
2-. Remember that we only use "sp" for textSize

Reply:
done!!!